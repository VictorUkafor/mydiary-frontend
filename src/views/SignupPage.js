import React from 'react';
import Layout from '../components/Layout';
import Signup from '../containers/Signup';

const LandingPage = (props) => (
  <div>
    <Layout page="signup" {...props}>
      <Signup {...props} />
    </Layout>
  </div>
);

export default LandingPage;
