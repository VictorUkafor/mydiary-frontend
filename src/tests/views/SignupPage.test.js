import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { cleanup } from 'react-testing-library';
import SignupPage from '../../views/SignupPage';

afterEach(cleanup);

test('<SignupPage />', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<SignupPage />);
  const result = renderer.getRenderOutput();

  expect(result.firstChild).toMatchSnapshot();
});
