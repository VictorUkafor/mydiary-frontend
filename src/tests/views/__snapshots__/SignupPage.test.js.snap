// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignupPage /> 1`] = `undefined`;
